src/App.tsx

```diff
@@ -1,9 +1,16 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = (moviesFromServer.filter((movie) => movie.description
+    .toLowerCase().trim().includes(query.toLowerCase().trim())
+      || movie.title.toLowerCase().trim().includes(query.toLowerCase().trim()))
+  );
```

Comment:
checklist p2
try to move the same logic to helper functions :)

Reply:
I don`t move it to helper, but I try to optimisate this one. What are you thinking?